relay_config/credentials.json

```diff
@@ -0,0 +1,5 @@
+{
+  "secret_key": "xWWsHUXG6RxHS9L7QWPrP0aDgVLKSa7vcE4Ypa-Vv44",
```

Comment:
We should definitely add this file to `.gitignore` and generate it at install time via `install.sh`.

Reply:
now generated at installation time